org.freedesktop.Geoclue.xml

```diff
@@ -10,6 +10,9 @@
     <method name="GetStatus">
       <arg name="status" type="i" direction="out"/>
     </method>
+    <method name="GnssDeleteAidingData">
```

Comment:
Is this part of the geoclue DBus API?  I couldn't find anything in Geoclue about this from a quick search, but maybe I missed something.  Seems like this is more a custom extension to expose some Android GNSS functionality, in which case I'm not sure it belongs here per-se.

Reply:
Where should I put it?

Reply:
If it is necessary to invoke this method via dbus, then this method could be exposed in a separate dbus interface (org.sailfishos.location.provider.Hybris perhaps).

Reply:
Why complicate the service with an additional interface dbus because of one additional command. Can I do without the additional dbus interface? Deleting data should not be separate from the main commands and is nothing else. This must be part of Geoclue

Reply:
Do agree that better not inject custom api to interface that tells it's a generic one. For another way, would SetOptions() work for this kind of things?

Also where is this getting called from?

Reply:
This can be used from any user application, such as the GNSS Test program. And it is a part of any GNSS receiver